src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -1140,6 +1140,12 @@
     <Field Name="InitializerExpressionOpt" Type="BoundExpression" Null="allow"/>
   </Node>
   
+    <Node Name="BoundTupleCreationExpression" Base="BoundExpression">
+    <!-- Non-null type is required for this node kind -->
+    <Field Name="Type" Type="TypeSymbol" Override="true" Null="disallow"/>
```

Comment:
I think that a tuple creation expression doesn't have a type. Or rather, there is probably a "raw" version without a type and a "cooked" version with a type where each element has been converted.